templates/careers/progression.html

```diff
@@ -1,32 +1,343 @@
+          </div>
+          <div class="p-careers-progression-carousel__slide--text" role="group" aria-label="slide 3 of 4">
+            <h2>Then choose to get more travel and focus on customer operations.</h2>
+          </div>
+          <div class="p-careers-progression-carousel__slide--text" role="group" aria-label="slide 4 of 4">
+            <h2>Then take on a role in tech leadership and management.</h2>
+          </div>
+        </div>
+      </div>
+      <div class="u-hide--large">
+        {% include 'careers/_careers-progression-slides.html' %}
+      </div>
+      <div class="p-careers-progression-carousel__controls">
+        <div>
+          <ul class="p-careers-progression-carousel__navigation">
+            <li>
+              <button type="button" aria-current="true" class="p-careers-progression-carousel__selector" tabindex="0" aria-current="true">
+                <span class="u-off-screen">Go to slide 1</span>
+              </button>
+            </li>
+            <li>
+              <button type="button" aria-current="false" class="p-careers-progression-carousel__selector" tabindex="1" aria-current="false">
+                <span class="u-off-screen">Go to slide 2</span>
+              </button>
+            </li>
+            <li>
+              <button type="button" aria-current="false" class="p-careers-progression-carousel__selector" tabindex="2" aria-current="false">
+                <span class="u-off-screen">Go to slide 3</span>
+              </button>
+            </li>
+            <li>
+              <button type="button" aria-current="false" class="p-careers-progression-carousel__selector" tabindex="3" aria-current="false">
+                <span class="u-off-screen">Go to slide 4</span>
+              </button>
+            </li>
+          </ul>
+        </div>
+        <div class="p-carousel-buttons__group">
+          <button type="button" class="p-careers-progression-carousel__previous">
+            <span class="u-off-screen">Previous slides</span>
+          </button>
+          <button type="button" class="p-careers-progression-carousel__next">
+            <span class="u-off-screen">Next slides</span>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="u-hide--medium u-hide--small">
+    {% include 'careers/_careers-progression-slides.html' %}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--careers-progression">
+  <div class="u-fixed-width" data-animation="u-animation--slide-from-bottom">
+    <p>Our mission in action</p>    
+  <h3 class="p-heading--2 p-strip--careers-progression u-no-padding--top">See what's new at Canonical</h3>
+  </div>
+  <div class="row u-extra-space" data-animation="u-animation--slide-from-bottom">
+    <div class="col-6 col-medium-3">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/646204f1-Elektrobit 1.png",
+        alt="EB corbos Linux with Elektrobit",
```

Comment:
Does this sentence describe the image to someone that can't see it?

Reply:
Well, I can remove it, but the a11y validator complains of missing `alt`